src/Kodify/BlogBundle/Resources/views/Post/List/index.html.twig

```diff
@@ -1,17 +1,34 @@
 {% extends 'KodifyBlogBundle::base.html.twig' %}
 
 {% block content %}
-
+    <div class="row">
+        <div class="pull-right">
+            <ul class="nav nav-pills">
+                <li role="presentation" {% if sort == 1 %}class="active"{% endif %}><a href="{{ url('sorted_home',{'sort': 1}) }}">Order by date</a></li>
+                <li role="presentation" {% if sort == 2 %}class="active"{% endif %}><a href="{{ url('sorted_home',{'sort': 2}) }}">Order by rating</a></li>
```

Comment:
Could you provide an easy alternative to specify the constants of the sort parameter of sorted_home url? (now as magic numbers)